pkg/api/types.go

```diff
@@ -156,8 +158,22 @@ type Config struct {
 
 	// Usage allows for properly shortcircuiting s2i logic when `s2i usage` is invoked
 	Usage bool
+
+	// Injections specifies a list source/destination folders that are injected to
+	// the container that runs assemble. All files are truncated after the
+	// assemble script runs, so they wont be present in the output image.
```

Comment:
the files **will** be present, but with size 0, right? That's an important distinction as the file names will be still available and might disclose information to an attacker.

BTW, do you know if after truncation it is possible to "recover" the truncated files by bypassing the file system? That would also be a security concern, as the file contents would still be there, but "hidden".